modular_skyrat/modules/pixel_shift/code/pixel_shift_mob.dm

```diff
@@ -0,0 +1,5 @@
+/mob/proc/add_pixel_shift_component()
+	return
+
+/mob/living/add_pixel_shift_component()
+	AddComponent(/datum/component/pixel_shift)
```

Comment:
Proc's not really necessary anymore, you can just do the following in the keybind proc
```dm
if(isliving(user.mob))
    user.mob.AddComponent(/datum/component/pixel_shift)
````

Reply:
The proc is there to allow blocking pixelshift on specific mobs. For example, a downstream would prefer to disable pixelshift on `silicon/ai`